end-to-end_sales-forecasting/sales_forecasting-end-to-end.ipynb

```diff
@@ -0,0 +1,2701 @@
+{
```

Comment:
<p>Do we need so much eda here? Might be better to get to using TruEra sooner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I removed most of the EDA in a prior iteration. The steps that remain are requisite for processing the raw data to a state that it can be modeled and ingested by TruEra. In the interest of this actually being an 'end to end' notebook, I'd like to leave the remaining steps in. </p>